index.ts

```diff
@@ -377,6 +377,11 @@ collect.set('isRepoMainSettings', [
 	'https://github.com/sindresorhus/refined-github/settings',
 ]);
 
+export const isUserSettings = (url: URL | HTMLAnchorElement | Location = location): boolean => getCleanPathname(url) === 'settings';
+collect.set('isUserSettings', [
+	'https://github.com/settings',
```

Comment:
This page doesn't exist, it's a redirect to `https://github.com/settings/profile` and this doesn't match this new check. I think the check should be `url.pathname.startsWith('/settings/)`

Reply:
I think this is why we never made that. We always made specific pages as we needed it

Reply:
What do you need this for @stefanbuck?

Reply:
Good shout. Addressed in 1bf7bdf

Reply:
@stefanbuck what is your use case?

Reply:
> @stefanbuck what is your use case?

I'm working on a new browser extension and I was thinking about adding the extension settings right into GitHub rather than using an extension settings dialog.

Reply:
Looking forward to seeing that! Are you planning to use [webext-options-sync](https://github.com/fregante/webext-options-sync)?

Reply:
`webext-options-sync` looks neat, so I will consider using it.